package.json

```diff
@@ -70,12 +70,14 @@
 		"@typescript-eslint/eslint-plugin": "^7.2.0",
 		"@typescript-eslint/parser": "^7.2.0",
 		"autoprefixer": "^10.4.19",
+		"cheerio": "^1.0.0-rc.12",
```

Comment:
Let's not add dependency unless it's necessary. Do we really need this cheerio?

Reply:
removed